### PR TITLE
Created Dockerfile for instant deployment

### PR DESCRIPTION
**Pull request by FIRSTB0SS**
### Since Docker is a Thing
I really wanted to run this amazing project on my server, but no dockerfile was present. So I made one, You can pull the image using
`docker pull firstboss/ai-debugger`

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+FROM node:16
```
Comment (DucretJe) [inline Dockerfile:1]:
```suggestion
FROM node:19
```

`package.json` requires v18, it seems to work also on v19
https://github.com/shobrook/adrenaline/blob/master/package.json#L56

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# where available (npm@5+)
+COPY package*.json ./
+
+RUN npm install
+# If you are building your code for production
+# RUN npm ci --only=production
+
+# Bundle app source
+COPY . .
+
```
Comment (DucretJe) [inline Dockerfile:17]:
```suggestion
USER node
```

root user should be avoided

**Comment by DucretJe** [comment]
Can we also update the readme to add the commands for docker?

File: Dockerfile (hunk 1 above)
Comment (FIRSTB0SS) [reply Dockerfile]: Yep

File: README.md [hunk 3]
```diff
@@ -18,6 +18,16 @@ To run locally, clone the repository and run the following:
 $ npm install
 $ npm run start-local
 ```
+# Using Docker
```
Comment (DucretJe) [inline README.md:21]:
```suggestion
### Running with Docker
```
Should be only one `#` per markdown

File: README.md [hunk 4]
```diff
@@ -18,6 +18,16 @@ To run locally, clone the repository and run the following:
...
+# Using Docker 
+
+> Quick deployment 
+
+``` $ docker run -d -p 3000:3000 firstboss/ai-debugger ```
+> access via ```http://<yourip>:3000```
+
++Build from source
+``` $ docker build -t adrenaline ```
+``` $ docker run -rm -d -p 3000:3000 adrenaline```
```
Comment (DucretJe) [inline README.md:30]:
```suggestion
```bash
$ docker build -t adrenaline .
$ docker run --rm -d -p 3000:3000 adrenaline
```

Let's keep consistency with the previous code blocks using `bash`
The registry you mention L25 doesn't look to be controlled by this repository, so I wouldn't trust it ;-)

```

**Review by DucretJe — COMMENTED** [review]
Sorry I'm a bit maniac, but thanks you for the fixes!

**Pull request closed without merging**